src/Microsoft.AspNet.Mvc.DataAnnotations/DataAnnotationsModelValidatorProvider.cs

```diff
@@ -45,7 +45,18 @@ public void GetValidators(ModelValidatorProviderContext context)
 
             foreach (var attribute in context.ValidatorMetadata.OfType<ValidationAttribute>())
```

Comment:
This is `Linq` YO.

I'd expect to see a `using` removed in each of these files :+1: